docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Collect a distributed trace - .NET
+description: A tutorial to collect distributed traces in .NET applications
+ms.topic: tutorial
+ms.date: 03/14/2021
+---
+
+# Collect a distributed trace
+
+**This article applies to: ✔️** .NET Core 5.0 and later versions **or** any .NET application using the 
```

Comment:
minor:
I'd try to word this to say that - This is applicable for all .NET versions, and then further clarify that - .NET 5 or higher, requires no extra dependency, older versions require installing SD.DS package.
This will avoid any potential misunderstanding that this is only a new thing and only works for newer .NET only.
(#1 question we received in otel as well!)

Reply:
Good feedback. I've update the 'applies to' guidance to indicate .NET Framework >= 4.5 and .Net Core >= 2.1. The need to include DiagnosticSource NuGet is embedded in the tutorial text but no longer mentioned here.